samples/subsys/logging/syst/sample.yaml

```diff
@@ -1,9 +1,19 @@
 sample:
   name: syst sample
+
+common:
+  arch_exclude: mips nios2 posix sparc
+  filter: not CONFIG_64BIT
+
 tests:
   sample.logger.syst.v1.deferred:
     extra_args: OVERLAY_CONFIG=overlay_deferred.conf
-    platform_allow: mps2_an385 qemu_x86 qemu_x86_64 sam_e70_xplained qemu_cortex_a53
+    integration_platforms:
+      - mps2_an385
+      - qemu_x86
+      - qemu_x86_64
```

Comment:
Is this a mistake, since you're filtering out 64-bit platforms?

Reply:
Good catch.